apps/dashboard/app/controllers/batch_connect/settings_controller.rb

```diff
@@ -0,0 +1,43 @@
+# The controller to manage BatchConnect saved settings
+class BatchConnect::SettingsController < ApplicationController
+  include BatchConnectConcern
+  include UserSettingStore
+
+  # GET /batch_connect/<app_token>/settings/<settings_name>
+  def show
+    set_app_groups
+    set_saved_settings
+
+    app_token = params[:token]
+    settings_name = params[:id]
```

Comment:
We should `permit` these params like we do in other controllers.

Reply:
My understanding was that `permit` was used when you use the parameters hash as a whole, passing it to a model or to another object for processing to filter unwanted parameters. I thought it was not needed in cases were the parameters were accessed directly.

Reply:
Added the permit to the controller